### PR TITLE
Update pluginconfig v12 to OMS agent 11012019

### PR DESCRIPTION
**Pull request by olga-mir**
```release-note
Upgrade OMS agent image to ciprod11012019
```

### CODE REVIEW

File: pluginconfig/pluginconfig-311.yaml [hunk 1]
```diff
@@ -301,7 +301,7 @@ versions:
       startup: quay.io/openshift-on-azure/ci-azure:latest
       sync: quay.io/openshift-on-azure/ci-azure:latest
       tlsProxy: quay.io/openshift-on-azure/ci-azure:latest
-      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod10182019
+      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11012019
```
Comment (ganga1980) [inline pluginconfig/pluginconfig-311.yaml]: Hi, @olga-mir , in addition to  agent image version change and there are other changes such as

- change from "deployments" to "replicasets"
- change the dockerProviderVersion from "7.0.0-5"  to "7.0.0-6"
- other changes as well

Can you please look at this PR  https://github.com/microsoft/OMS-docker/pull/335/files and make the applicable updates  to this PR.

Comment (olga-mir) [reply pluginconfig/pluginconfig-311.yaml]: Hi @ganga1980 thanks for reviewing. I've updated PR to include these changes.

Comment (ganga1980) [reply pluginconfig/pluginconfig-311.yaml]: @olga-mir, changes looks good. I see the cluster in the ux and all the data getting ingested.

Comment (ganga1980) [reply pluginconfig/pluginconfig-311.yaml]: lgtm

**Comment by olga-mir** [comment]
/hold

hold until v13 is merged

**Comment by jim-minter** [comment]
@olga-mir v13 is open for business, please move this PR's changes from v12 to v13 and we'll get it merged.

**Comment by olga-mir** [comment]
Closing this PR, I've created v13 PR from a clean branch: https://github.com/openshift/openshift-azure/pull/2077

**Pull request closed without merging**